recipes/cpu_features/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cpu_features" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "2feca41fe51d32f53d22d7fb75e4b6ef5cab70db0ff2620259bbfc8f1a9cd887" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.src.tar.xz
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+# no host:
+# no   - googletest
+# no run:
+
+test:
+  commands:
+    - list_cpu_features
+    - if not exist %LIBRARY_INC%\\cpu_features\\cpu_features_macros.h         exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\cpu_features.lib                            exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\cmake\\CpuFeatures\\CpuFeaturesConfig.cmake exit 1  # [win]
+    - test -f $PREFIX/include/cpu_features/cpu_features_macros.h                      # [unix]
+    - test -f $PREFIX/lib/libcpu_features.a                                           # [unix]
+    - test -f $PREFIX/lib/cmake/CpuFeatures/CpuFeaturesConfig.cmake                   # [unix]
+
+about:
+  home: https://github.com/google/cpu_features
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'A cross platform C99 library to get cpu features at runtime'
+  dev_url: https://github.com/google/cpu_features
+
+extra:
+  recipe-maintainers:
+    - anton-malakhov
+    - aregm
```

Comment:
@aregm are you happy being listed as a maintainer of this recipe?

Reply:
> @aregm are you happy being listed as a maintainer of this recipe?

Yes, confirmed! Thanks.